azuredeploy.json

```diff
@@ -706,6 +786,13 @@
             "autoscaleVmCount": "[parameters('autoscaleVmCount')]",
             "autoscaleVmSku": "[parameters('autoscaleVmSku')]",
             "azureBackupSwitch": "[parameters( 'azureBackupSwitch')]",
+            "azureSearchHostingMode": "[parameters( 'azureSearchHostingMode')]",
+            "azureSearchName": "[concat('azure-search-',variables('resourceprefix'))]",
+            "azureSearchNameHost": "[concat('azure-search-',variables('resourceprefix'),'.search.windows.net')]",
+            "azureSearchPartitionCount": "[parameters( 'azureSearchPartitionCount')]",
```

Comment:
Nitpicking, but any reason why there's a space between the opening paren and the first param? We should have fixed all these in the first place, and let's not add to that any more...